src/content/doc-surrealdb/querying/graphql/http.mdx

```diff
@@ -34,10 +34,23 @@ $env:SURREAL_EXPERIMENTAL_GRAPHQL=true surreal start --log debug --user root --p
 </TabItem>
 </Tabs>
 
+In order to allow querying the created table using GraphQL, you will need to explicitly enable graphql using the [`DEFINE CONFIG graphql`](/docs/surrealql/statements/define/config) statement. This will allow you to query the table using GraphQL on a per-database basis.
+
 > [!WARNING]
 > By running SurrealDB with the GraphQL module enabled, you are opting into an experimental feature. While the GraphQL module is fully functional, it is still considered experimental and may not be as stable as the core SurrealQL module which means we cannot guarantee that it will provide the same security guarantees. It is not recommended for production use. We welcome your feedback and contributions to help improve the feature and make it more robust.
 
 
+## `POST /sql`
+
+To use the GraphQL API, you first need to enable it using the `DEFINE CONFIG graphql` statement. This will allow you to query the table using GraphQL on a per-database basis.
```

Comment:
```suggestion
To use the GraphQL API, you first need to enable it using the `DEFINE CONFIG` statement. This will allow you to query the table using GraphQL on a per-database basis.
```